src/Workspaces/CSharp/Portable/Indentation/CSharpIndentationService.Indenter.cs

```diff
@@ -73,6 +74,37 @@ private static IndentationResult GetIndentationBasedOnToken(Indenter indenter, S
             Contract.ThrowIfNull(indenter.Tree);
             Contract.ThrowIfTrue(token.Kind() == SyntaxKind.None);
 
+            var sourceText = indenter.LineToBeIndented.Text;
+            RoslynDebug.AssertNotNull(sourceText);
+
+            // case: """$$
+            //       """
+            if (token.IsKind(SyntaxKind.MultiLineRawStringLiteralToken))
+            {
+                var endLine = sourceText.Lines.GetLineFromPosition(token.Span.End);
+                var nonWhitespaceOffset = endLine.GetFirstNonWhitespaceOffset();
+                Contract.ThrowIfNull(nonWhitespaceOffset);
+                return new IndentationResult(indenter.LineToBeIndented.Start, nonWhitespaceOffset.Value);
+            }
+
+            // case 1: $"""$$
+            //          """
+            // case 2: $"""
+            //          text$$
+            //          """
```

Comment:
```suggestion
            // case 1: $"""$$
            //         """
            // case 2: $"""
            //         text$$
            //         """
```

Reply:
the indenter isn't setting where the delimiter goes here.  Instead, it's stating where the caret should go when enter is typed *above* teh delimeter.  In that case, the caret is indented the same amount as the final string.